test/unit/mocked_data/test_get_bgp_neighbors/use-send-state-pr209/get-bgp-neighbor-information.xml

```diff
@@ -1,10 +1,10 @@
 <bgp-information>
-    <bgp-peer junos:style="detail">
+    <bgp-peerstyle="detail">
```

Comment:
Erm - too much removed: make sure you leave the space, i.e., `bgp-peer style="detail"` etc.